frontend/src/components/JoinContainerSeparator/JoinContainerSeparator.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import { describe, it, expect } from 'vitest';
+import JoinContainerSeparator from './JoinContainerSeparator';
+
+describe('JoinContainerSeparator', () => {
```

Comment:
Are these new test files you added ?

Reply:
Yes, they are. just to keep sonarcloud happy to get over that 80% threshold since I touched on some files that did not have tests